core/src/com/unciv/JsonParser.kt

```diff
@@ -0,0 +1,19 @@
+package com.unciv
+
+import com.badlogic.gdx.Gdx
+import com.badlogic.gdx.utils.Json
+
+interface JsonParser {
```

Comment:
Completely agree with separating this to another class
But remember what I said about interfaces...?
This is a good example - your "mock" is in fact exactly the same as your Impl(), so this should probably just be a regular class

Reply:
Ok will fix